base/utils/CursorManager.js

```diff
@@ -0,0 +1,96 @@
+const WIGGLE_X1 = -4;
+const WIGGLE_Y1 = +4;
+
+const WIGGLE_X2 = -8;
+const WIGGLE_Y2 = 0;
+
+const POINT_X = -6;
+const POINT_Y = 6;
+
+const TWEEN_TIME = Phaser.Timer.QUARTER >> 1;
+const MOVE_TIME = Phaser.Timer.QUARTER >> 1;
+
+export class CursorManager{
+    constructor(game){
+        this.game = game;
+
+        this.group = this.game.add.group();
+        this.group.alpha = 1;
+        this.group.x = 0;
+        this.group.y = 0;
+        this.cursor = this.group.create(0, 0, "cursor");
+        this.active_tween = null;
+        this.current_tween = null;
+        this.cursor.default_pos = {x: 0, y: 0};
+    }
+
+    clear_tweens(){
+        if(this.active_tween) this.game.tweens.remove(this.active_tween);
+        this.active_tween = null;
+
+        this.cursor.x = this.cursor.default_pos.x;
+        this.cursor.y = this.cursor.default_pos.y;
+    }
+
+    init_tween(type){
+        this.clear_tweens();
+
+        switch(type){
+            case "wiggle":
```

Comment:
Try to put these key names into variables. At least in consts variables at the top of the file.